zerver/openapi/zulip.yaml

```diff
@@ -12014,6 +12014,147 @@ paths:
+
+        **Changes**: In Zulip 8.0 (feature level 209), added support for non-admin
+        users to use this endpoint. Previously, it was restricted to administrators
+        only.
+
+        In Zulip 6.0 (feature level 126), the `invite_expires_in_days`
+        parameter was removed and replaced by `invite_expires_in_minutes`.
+
+        In Zulip 5.0 (feature level 117), added support for passing `null` as
+        the `invite_expires_in_days` parameter to request an invitation that never
+        expires.
+
+        In Zulip 5.0 (feature level 96), the `invite_expires_in_days` parameter was
+        added which specified the number of days before the invitation would expire.
+      requestBody:
+        required: false
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                invite_expires_in_minutes:
+                  description: |
+                    The number of minutes before the invitation will expire. If `null`, the
+                    invitation will never expire. If unspecified, the server will use a default
+                    value (`INVITATION_LINK_VALIDITY_MINUTES`) for when the invitation will expire.
+
+                    **Changes**: New in Zulip 6.0 (feature level 126). Previously, there was an
+                    `invite_expires_in_days` parameter, which specified the duration in days instead
+                    of minutes.
+                  type: integer
+                  nullable: true
+                  example: 14400
+                invite_as:
+                  description: |
+                    The [organization-level role](/api/roles-and-permissions) of the user that is
+                    created when the invitation is accepted.
+
+                    Users can only create invitation links for
+                    [roles with equal or stricter restrictions](/api/roles-and-permissions#permission-levels)
+                    as their own. For example, a moderator cannot invite someone to be an owner
+                    or administrator, but they can invite them to be a moderator or member.
+
+                    **Changes**: In Zulip 4.0 (feature level 61), added support for inviting
+                    users as moderators.
+                  type: integer
+                  enum:
+                    - 100
+                    - 200
+                    - 300
+                    - 400
+                    - 600
+                  default: 400
+                  example: 600
+                stream_ids:
+                  description: |
+                    A list containing the [IDs of the streams](/api/get-stream-id) that the
+                    newly created user will be automatically subscribed to if the invitation
+                    is accepted. If unspecified, it will be set to empty list. If the list is
+                    empty, then the new user will not be subscribed to any streams.
```

Comment:
@sahil839 @mateuszmandera is this an accurate description of how the `stream_ids` parameter works?

Reply:
Yes, this is correct. We do not add users to any stream if the `stream_ids` list is explicitly empty.

Though there will be some change in this behavior where if a user is not allowed to subscribe others, the new user will be added to default streams in #25739.